karate-tests/src/test/java/patients/patientAccess/updatePatient.feature

```diff
@@ -1,4 +1,3 @@
-@contactTest
```

Comment:
just curious, why is this deleted please?

Reply:
This annotation was added  for local debugging purpose in the previous pr.  It was unintentionally left in the code and is no longer needed, so I am removing it in this PR.